src/care/algorithm_impl.h

```diff
@@ -1515,21 +1515,79 @@ CARE_INLINE void ArrayCopy(RAJA::seq_exec,
    } CARE_SEQUENTIAL_LOOP_END
 }
 
+/************************************************************************
+ * Function  : ArrayCopy
+ * Author(s) : Peter Robinson
+ * Purpose   : Duplicates a ManagedArray.
+ * ************************************************************************/
+template <typename T>
+CARE_INLINE care::host_device_ptr<T> ArrayDup(care::host_device_ptr<const T> from, int len)
+{
+   return ArrayDup<T>(RAJAExec {}, from, len);
+}
+
+/************************************************************************
+ * Function  : ArrayCopy
```

Comment:
Documentation: change "ArrayCopy" to "ArrayDup" to match the function name.

Reply:
Fixed.